### PR TITLE
Remove old APIs and browser compatibility

### PR DESCRIPTION
**Pull request by jugarrit**

### CODE REVIEW

File: src/wrappers/node.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
 if (!global.FormData) {
```
Comment (lemieux) [inline src/wrappers/node.js]: Not sure we need the wrapper anymore, that piece of code should just go in smooch.js

File: src/api/appUsers.js [hunk 2]
```diff
@@ -97,61 +79,6 @@ Object.assign(AppUsersApi.prototype, {
...
-    /**
-     * Update the push notification token for a given app user's device
-     * @memberof AppUsersApi.prototype
-     * @method updatePushToken
-     * @param  {string} userId   - a user id
-     * @param  {string} deviceId - a device id
-     * @param  {string} token    - a push notification token
-     * @return {APIResponse}
-     */
-    updatePushToken: smoochMethod({
```
Comment (mspensieri) [inline src/api/appUsers.js]: 🤔  this method wasn't in the readme?

Comment (jugarrit) [reply src/api/appUsers.js]: Nope!

File: src/smooch.js [hunk 3]
```diff
@@ -15,9 +15,13 @@ import { decode } from 'jsonwebtoken';
...
 
-export const SERVICE_URL = 'https://api.smooch.io/v1';
+const SERVICE_URL = 'https://api.smooch.io/v1';
 
-export class Smooch {
+if (!global.FormData) {
+    global.FormData = require('form-data');
+}
+
+class Smooch {
```
Comment (lemieux) [inline src/smooch.js]: I think you could use `export default class Smooch` instead of mixing es6 import/export with `module.exports`

Comment (jugarrit) [reply src/smooch.js]: As discussed offline, classes exported this way do not `require` the same as a CommonJS module.

**Pull request merged**